app/src/main/java/com/android/doctorapp/ui/userdashboard/userfragment/UserRequestFragment.kt

```diff
@@ -38,10 +39,18 @@ class UserRequestFragment :
     lateinit var adapter: BookingAppointmentsAdapter
     private val myCalender: Calendar = Calendar.getInstance()
 
+
+    private var currentPage = PaginationScrollListener.PAGE_START
+    private var isLastPage = false
+    private var totalPage = 10
```

Comment:
Why total page is 10

Reply:
Based on total page is check that if list Size  less than 10 so its last Page otherwise is showing the Progress.
 if (list.size >= totalPage) {
                adapter.addLoading()
            } else {
                isLastPage = true
            }